{{cookiecutter.project_name}}/CONTRIBUTING.rst

```diff
@@ -0,0 +1,127 @@
+.. highlight:: shell
```

Comment:
IMHO this file does not add much as a template. Some of the instructions are too basic (how to create a branch and submit a pull request). Others depends a lot on the kind of lib/project we are publishing (like having a _Fix Bugs_ and _Implement Features_ section while we usually use our JIRA to report the issues).

The section _Getting Started_ has valuable info, but I would add it to `README` file.

I would like the opinion of @nicoddemus on that.

Reply:
I agree

Reply:
`CONTRIBUTING` is kind of a standard for open source projects, additionally, Github recommends to have a `CONTRIBUTING` file, because it brings some "nice UI features" as mentioned on [`Github documentation`](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)